robottelo/cli/user.py

```diff
@@ -93,3 +94,14 @@ def ssh_keys_info(cls, options=None):
         """
         cls.command_sub = 'ssh-keys info'
         return cls.execute(cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def personal_access_token(cls, action=None, options=None):
```

Comment:
```suggestion
    def access_token(cls, action=None, options=None):
```
This was my original suggestion

Reply:
My bad. 
Changes done